tools/clang/lib/Serialization/ASTReader.cpp

```diff
@@ -201,11 +201,16 @@ static bool checkLanguageOptions(const LangOptions &LangOpts,
   if (!AllowCompatibleDifferences)                            \
     LANGOPT(Name, Bits, Default, Description)
 
+#define COMPATIBLE_LANGOPT_BOOL(Name, Default, Description)  \
+  if (!AllowCompatibleDifferences)                            \
+    LANGOPT_BOOL(Name, Default, Description)
+
 #define COMPATIBLE_ENUM_LANGOPT(Name, Bits, Default, Description)  \
   if (!AllowCompatibleDifferences)                                 \
     ENUM_LANGOPT(Name, Bits, Default, Description)
 
 #define BENIGN_LANGOPT(Name, Bits, Default, Description)
+#define BENIGN_LANGOPT_BOOL(Name, Default, Description)
```

Comment:
Another omitted LANGOPT_BOOL define and I'm starting to think I must be missing something

Reply:
Added default define for LANGOPT_BOOL.